components/discord_webhook/actions/send-message-advanced/send-message-advanced.mjs

```diff
@@ -46,13 +46,21 @@ export default {
       content += `\n\n${this.getSentViaPipedreamText()}`;
     }
 
-    // No interesting data is returned from Discord
-    await this.discordWebhook.sendMessage({
+    const params = {
       avatarURL,
       embeds,
       content,
       threadID,
       username,
-    });
+    };
+
+    try {
+      // No interesting data is returned from Discord
+      await this.discordWebhook.sendMessage(params);
+      $.export("$summary", "Message sent successfully");
+    } catch (err) {
+      $.export("$summary", `${err}`);
```

Comment:
I think the [`$summary`](https://pipedream.com/docs/components/guidelines/#guidelines-patterns) should use plain language to summarize what happened. E.g., "Failed to send the message. See the `unsent` [export](https://pipedream.com/docs/components/api/#returning-data-from-steps) for the unsent message."?  
Typically a `$summary` isn't emitted when an action fails.

Reply:
Addressed